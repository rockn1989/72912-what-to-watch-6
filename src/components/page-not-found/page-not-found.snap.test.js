import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import PageNotFound from './page-not-found';

it(`Should PageNotFound render correctly`, () => {
  const history = createMemoryHistory();
  const {container} = render(
      <Router history={history}>
        <PageNotFound />
      </Router>
  );
  expect(container).toMatchSnapshot();
});
