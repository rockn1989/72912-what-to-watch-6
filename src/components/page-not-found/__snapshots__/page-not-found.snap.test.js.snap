// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PageNotFound render correctly 1`] = `
<div>
  <div
    class="page-not-found"
  >
    <h1>
      Страница не найдена
    </h1>
    <a
      href="/"
    >
      Вернуться на главную
    </a>
  </div>
</div>
`;
